### ISSUE TITLE
How to reduce excessive linespam when replying to comments directly?

### ISSUE THREAD

**Issue by pythonInRelay** [vector-im/element-web#18078]
#### Description

Using a browser, when replying specifically/directly to a comment, replies by another user (not the user who made the initial comment) include a quotation of the first user's comment and replies. This is unnecessarily excessive and results in a lot of linespam. Please see the included image. Something similar to how Slack handles replies should be used instead.

![image](https://i.postimg.cc/SQ7rhSw-n/elementtest.png)

# Proposed Fix/Change

![icon](https://i.postimg.cc/7hfYmYzK/icon.png) --Username1-- First Comment
>![icon](https://i.postimg.cc/7hfYmYzK/icon.png) --Username1-- Second Comment
>![icon2](https://i.postimg.cc/wxC70LWJ/icon.png) --Username2-- First Comment

The above example is completely readable and follows on the discussion. Alternatively, after 3+ replies (would be nice if the admin could choose) only the last 3 replies are shown to avoid again, more linespam.

**Comment by SimonBrandner** [comment]
Replies got recently completely changed from the UI standpoint, the changes should land in the coming weeks. Could you please try out develop.element.io and see if the new replies there look alright to you? If not, could you please update the issue with comments about the new replies?

**Comment by pythonInRelay** [comment]
Hey there, thanks for the reply.

It's not really any better on the development version. As I said in the proposed fix/change. Something similar to Slack would be really nice. There's a little bit less empty space on the dev version but otherwise it could be tuned a lot more.

**Comment by SimonBrandner** [comment]
So, you would prefer something Discordy, right?

![Screenshot_20210722_182445](https://user-images.githubusercontent.com/25768714/126674146-c954f65a-11c9-4efe-93fa-a92001431ae9.png)

**Comment by pythonInRelay** [comment]
Well no, but kind of anything is better than what exists at the moment :( I think most of us would appreciate something like what Slack has, including the profile pictures.

**Comment by ShadowJonathan** [comment]
Just for brevity, could you post a screenshot of how slack does it?

**Comment by pythonInRelay** [comment]
> 
> 
> Just for brevity, could you post a screenshot of how slack does it?

Sure thing. Here you go:
![slack](https://i.postimg.cc/7LGV0145/slack.png)

**Comment by SimonBrandner** [comment]
So you're asking for threading support?

**Comment by pythonInRelay** [comment]
Yes that would be great. I think it would also be fine if you wanted to do it differently. For example, just collapse all replies when the user clicks "x replies". The main thing is that in channels with a huge amount of people it can start to get very cumbersome currently.

**Comment by ShadowJonathan** [comment]
I dont think this is a good idea, i've already spoken about this at https://github.com/vector-im/element-web/issues/18171#issuecomment-884744666, in that it changes a core UX part of Element for a worse turn.

It all depends on Product though, if they find this a good idea, i'd let them, but I just wanna note that seperating replies and threading is a good idea, combining them results in the mess of "banishing replies to the sidebar", like i said in that comment.

**Comment by pythonInRelay** [comment]
I definitely don't want to "bash comments to the sidebar". More like I said, in that you click "x replies" and it expands the replies below. Rather than having them in a sidebar, I never liked that about Slack either. Which is why I said specifically it would be nice to have it this way and I provided a screenshot of how Slack collapses direct replies. :)
